lib/src/commons/BrowserUtils.js

```diff
@@ -277,8 +277,8 @@ var BrowserUtils;
         chillOut();
```

Comment:
check if we still need it with v5

Reply:
removed the one before isExist left the one after switch